conf/profile.conf.js

```diff
@@ -171,6 +171,12 @@ exports.config = {
       '*': {
         '*': {
           browserName: 'chrome',
+          remoteWebDriverOptions: {
```

Comment:
maximized: false ?

Reply:
The default setting "maximized: true" does not apply to chromeHeadless. It is applied for: firefox, ie, edge, safari, chrome, chromium.